addon/flatten-keys.js

```diff
@@ -1,13 +1,9 @@
 import isComputed from './is-computed';
+import { getComputedData } from './-build-computed';
 
 function flattenKey(key, flattenedKeys) {
   if (isComputed(key)) {
-    let dependentKeys = key._dependentKeys;
-    if (dependentKeys === undefined) {
-      // when there are no keys (raw)
-      return;
-    }
-
+    let { dependentKeys } = getComputedData(key);
```

Comment:
Is there no falsy case anymore?

Reply:
Correct. The falsy case previously existed because [this](https://github.com/emberjs/ember.js/blob/be4f59aac70e0652127c04186be0318f18ee562a/packages/%40ember/-internals/metal/lib/computed.ts#L299-L301) prevented `_dependentKeys` from being set if there were zero of them, but I didn't see a reason to do that in the tracking we do here, so in that case we just [save off the empty array](https://github.com/kellyselden/ember-macro-helpers/pull/258/files#diff-071332462d723c81cd5be206b3ed6402R56).